doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -192,6 +192,10 @@ Bluetooth samples
 
   * The Bluetooth 3-wire coex sample because of the removal of the 3-wire implementation.
 
+* :ref:`peripheral_hids_mouse` sample:
+
+  * :kconfig:option:`CONFIG_BT_SMP` is included when `CONFIG_BT_HIDS_SECURITY_ENABLED` is enabled.
```

Comment:
```suggestion
  * The :kconfig:option:`CONFIG_BT_SMP` Kconfig option is included when ``CONFIG_BT_HIDS_SECURITY_ENABLED`` is enabled.
```
Is that latter also a Kconfig option? At least it is not introduced in the Kconfig documentation or in the sample doc. Sounds a bit weird as something called ... _ENABLED is enabled. Should we rather say that it is selected in the Kconfig file or something like that?